source-repo-scripts/bump_dependency.bash

```diff
@@ -187,54 +195,53 @@ commitAndPR() {
     return
   fi
 
-  local LIB=${CURRENT_BRANCH#bump_${COLLECTION}}
-  local LIB=${LIB#_}
-  local LIB=": ${LIB}"
-
   echo -e "${GREEN_BG}${REPO}: Commit ${REPO} and open PR? (y/n)${DEFAULT_BG}"
   read CONTINUE
   if [ "$CONTINUE" = "y" ]; then
-    git commit -sam"${COMMIT_MSG} ${LIB}"
+    git commit -sam"${COMMIT_MSG}
```

Comment:
pull requests to gazebodistro and homebrew-simulation should add the name of the package being edited at the start of the PR title and commit message

see the change in title for https://github.com/osrf/homebrew-simulation/pull/3008 for example

Reply:
added in https://github.com/gazebo-tooling/release-tools/pull/1302/commits/9fe5b56f0d485245440aef0a9262438bc3517aaa